src/Explorer/Controls/Settings/SettingsSubComponents/SubSettingsComponent.tsx

```diff
@@ -0,0 +1,443 @@
+    if (event.charCode === Constants.KeyCodes.Space || event.charCode === Constants.KeyCodes.Enter) {
+      event.stopPropagation();
+      event.preventDefault();
+      this.onGeographyLabelClick();
+    }
+  };
+
+  private onGeographyLabelClick = (): void => {
+    this.props.onGeoSpatialConfigTypeChange(GeospatialConfigType.Geography);
+  };
+
+  private onGeometryKeyPress = (event: React.KeyboardEvent<HTMLSpanElement>): void => {
+    if (event.charCode === Constants.KeyCodes.Space || event.charCode === Constants.KeyCodes.Enter) {
+      event.stopPropagation();
+      event.preventDefault();
+      this.onGeometryLabelClick();
+    }
+  };
+
+  private onGeometryLabelClick = (): void => {
+    this.props.onGeoSpatialConfigTypeChange(GeospatialConfigType.Geometry);
+  };
+
+  private onAnalyticalStorageTtlOnNoDefaultKeyPress = (event: React.KeyboardEvent<HTMLSpanElement>): void => {
+    if (event.charCode === Constants.KeyCodes.Space || event.charCode === Constants.KeyCodes.Enter) {
+      event.stopPropagation();
+      event.preventDefault();
+      this.onAnalyticalStorageTtlOnNoDefaultLabelClick();
+    }
+  };
+
+  private onAnalyticalStorageTtlOnNoDefaultLabelClick = (): void => {
+    this.props.onAnalyticalStorageTtlSelectionChange(TtlType.OnNoDefault);
+  };
+
+  private onAnalyticalStorageTtlOnKeyPress = (event: React.KeyboardEvent<HTMLSpanElement>): void => {
+    if (event.charCode === Constants.KeyCodes.Space || event.charCode === Constants.KeyCodes.Enter) {
+      event.stopPropagation();
+      event.preventDefault();
+      this.onAnalyticalStorageTtlOnLabelClick();
+    }
+  };
+
+  private onAnalyticalStorageTtlOnLabelClick = (): void => {
+    this.props.onAnalyticalStorageTtlSelectionChange(TtlType.On);
+  };
+
+  private onChangeFeedPolicyOffKeyPress = (event: React.KeyboardEvent<HTMLSpanElement>): void => {
+    if (event.charCode === Constants.KeyCodes.Space || event.charCode === Constants.KeyCodes.Enter) {
+      event.stopPropagation();
+      event.preventDefault();
+      this.onChangeFeedPolicyOffLabelClick();
+    }
+  };
+
+  private onChangeFeedPolicyOffLabelClick = (): void => {
+    this.props.onChangeFeedPolicyChange(ChangeFeedPolicyState.Off);
+  };
+
+  private onChangeFeedPolicyOnKeyPress = (event: React.KeyboardEvent<HTMLSpanElement>): void => {
+    if (event.charCode === Constants.KeyCodes.Space || event.charCode === Constants.KeyCodes.Enter) {
+      event.stopPropagation();
+      event.preventDefault();
+      this.onChangeFeedPolicyOnLabelClick();
+    }
+  };
+
+  private onChangeFeedPolicyOnLabelClick = (): void => {
+    this.props.onChangeFeedPolicyChange(ChangeFeedPolicyState.On);
+  };
+
+  private getTtlComponent = (): JSX.Element => {
+    return (
+      <Stack {...titleAndInputStackProps}>
+        <Text>Time to Live</Text>
+        <div className="tabs" id="timeToLive" aria-label="Time to Live" role="radiogroup">
+          <div className="tab">
+            <Label
```

Comment:
Same comment about reusing existing multi-choice component if you can.
Otherwise, if it's similar to the previous one, consider creating a new multi-choice component. Add aria-label. Inside that component, consider setting the correct tabIndex (0, 1, 2, etc).

Reply:
The radio buttons dont give the intended styling effect that is currently present AccessibleELement doesn't play well with the tabbing either. will create a new component which can have N choices and accept its effects (onClick, onPress) as an array.

Reply:
Just synced with Jessica, we decided to move all these elements to vertical radio buttons to be consistent.